app/test_main.py

```diff
@@ -1,3 +1,62 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAgeConversion:
+    @pytest.mark.parametrize("cat_age, dog_age, expected", [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (-1, 10, [0, 0]),
+        (0, 10, [0, 0]),
+        (5, -1, [0, 0]),
+        (5, 0, [0, 0]),
+        (1000, 1000, [246, 197]),
+    ], ids=[
+        "Both zero age",
+        "Both small age",
+        "Both middle age",
+        "Both middle age",
+        "Both middle age",
+        "Both middle age",
```

Comment:
Add more descriptive comments such as `both cats and dogs years equals 1 human year` etc